tests/components/waze_travel_time/test_config_flow.py

```diff
@@ -138,7 +139,7 @@ async def test_dupe(hass):
 
 
 @pytest.mark.usefixtures("invalidate_config_entry")
-async def test_invalid_config_entry(hass, caplog):
+async def test_invalid_config_entry(hass: HomeAssistant, caplog) -> None:
```

Comment:
This might fail black
```suggestion
async def test_invalid_config_entry(hass: HomeAssistant, caplog: pytest.LogCaptureFixture) -> None:
```